server/fleet/mdm.go

```diff
@@ -812,6 +812,46 @@ type CAConfigAsset struct {
 	Type  CAConfigAssetType `db:"type"`
 }
 
+type CAType string
```

Comment:
Mind if we move the CA stuff into server/fleet/certificate_authorities.go? The main reason being we are not treating these as their own resource now, similar to server/fleet/scripts.go.

Reply:
this is where I've placed some of the other structs on the branch I'm working in https://github.com/fleetdm/fleet/blob/feat-api-get-cert-authorities-endpoint/server/fleet/certificate_authorities.go.